src/System.Net.HttpListener/src/System/Net/HttpResponseStream.cs

```diff
@@ -40,6 +40,19 @@ public override IAsyncResult BeginRead(byte[] buffer, int offset, int size, Asyn
         }
 
         public override int EndRead(IAsyncResult asyncResult) => throw new InvalidOperationException(SR.net_writeonlystream);
+        
+        public new Task<int> ReadAsync(Byte[] buffer, int offset, int count)
```

Comment:
You should use override since you are override a virtual method.  You only use 'virtual' on the base class virtual method.  You use 'override' on all derived classes.